### DISCUSSION TITLE
Make kitty paint entire terminal

### DISCUSSION THREAD

**Discussion by TH3-S4LM0N** [kovidgoyal/kitty#5460] [Q&A] [answered]
Is it possible to make kitty actually use the entire terminal space. Aka not have spaces of terminal showing through like so: ![friends image host](https://salmon-likes.catboys.win/MP9Xr9kL.png)
The size is changeable with font size, but it doesn't go away, just gets smaller. How do i make (at least tui) render over the entire terminal?

**Comment by page-down** [comment, marked as answer]
You need to make the window size a multiple of the cell size.
I'm guessing you're using a tiled window manager, which is likely difficult to do.
The terminal does not have partial cells, so window margins will appear.
